charts/images.yaml

```diff
@@ -51,11 +39,22 @@ images:
   repository: quay.io/k8scsi/csi-snapshotter
   tag: v1.2.2
   targetVersion: ">= 1.14"
+- name: csi-snapshotter
+  sourceRepository: https://github.com/kubernetes-csi/external-snapshotter
+  repository: quay.io/k8scsi/csi-snapshotter
+  tag: v2.1.1
```

Comment:
As stated in  #72, the v2 of external-snapshotter has incompatible changes - it features a new snapshot-controller which also needs to be deployed. You could check https://github.com/gardener/gardener-extension-provider-gcp/pull/77 for more details.